src/core/lib/iomgr/cfstream_handle.cc

```diff
@@ -94,8 +105,14 @@ void CFStreamHandle::WriteCallback(CFWriteStreamRef stream,
       handle->write_event_.SetReady();
       break;
     case kCFStreamEventErrorOccurred:
-      handle->open_event_.SetReady();
-      handle->write_event_.SetReady();
+      stream_error = CFWriteStreamCopyError(stream);
+      error = grpc_error_set_int(
```

Comment:
this error does not seem to get unreffed anywhere. this would a memory leak

Reply:
fixed.